dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/AnalyticEventTypeHandlerManager.java

```diff
@@ -0,0 +1,22 @@
+package com.dotcms.rest.api.v1.experiment;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum AnalyticEventTypeHandlerManager {
```

Comment:
Rename to AnalyticsEventTypeHandlerManager

Reply:
I think it could EventHandlerManager